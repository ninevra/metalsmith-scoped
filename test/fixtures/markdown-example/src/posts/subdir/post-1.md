*Another* markdown file, in a subdirectory this time.
