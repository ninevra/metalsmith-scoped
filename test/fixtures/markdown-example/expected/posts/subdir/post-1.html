<p><em>Another</em> markdown file, in a subdirectory this time.</p>
